src/hatti/views/map.cljs

```diff
@@ -476,7 +429,7 @@
         (let [with-suffix #(if-not (om/get-state owner :expanded) %
                                    (str % " leaflet-control-layers-expanded"))]
           (html
-           [:div.leaflet-left.leaflet-bottom {:style {:margin-bottom "148px"}}
+           [:div.leaflet-left.leaflet-bottom {:style {:margin-bottom "190px"}}
```

Comment:
this is old code, but this definitely should be moved to stylesheets and removed from here